Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -157,20 +158,56 @@ public extension UIConfig {
         public var messageContentView: ChatMessageContentView<ExtraData>.Type = ChatMessageContentView<ExtraData>.self
         public var messageContentSubviews = MessageContentViewSubviews()
         public var messageActionsSubviews = MessageActionsSubviews()
+        public var messageReactions = MessageReactions()
     }
 
     struct MessageActionsSubviews {
+        public var actionsView: MessageActionsView<ExtraData>.Type =
+            MessageActionsView<ExtraData>.self
+        public var actionButton: MessageActionsView<ExtraData>.ActionButton.Type =
+            MessageActionsView<ExtraData>.ActionButton.self
+    }
+
+    struct MessageReactions {
+        public var reactionsBubbleView: ChatMessageReactionsBubbleView<ExtraData>.Type =
+            ChatMessageDefaultReactionsBubbleView<ExtraData>.self
         public var reactionsView: ChatMessageReactionsView<ExtraData>.Type = ChatMessageReactionsView<ExtraData>.self
+        public var reactionItemView: ChatMessageReactionsView<ExtraData>.ItemView.Type =
+            ChatMessageReactionsView<ExtraData>.ItemView.self
+
         public var availableReactions: [MessageReactionType] = [
             .init(rawValue: "like"),
             .init(rawValue: "haha"),
-            .init(rawValue: "facepalm"),
-            .init(rawValue: "roar")
+            .init(rawValue: "thumbsup"),
+            .init(rawValue: "thumbsdown"),
+            .init(rawValue: "wut")
         ]
```

Comment:
Should we unify the way we specify reaction somewhere? Maybe even using a protocol such that users can add their own custom data?
```swift
protocol ReactionType { 
  let stringValue: String
  let smallIcon: UIImage
  let largeIcon: UIImage
}

UIConfig.default.availableReactions = [Reaction1, Reaction2, ...]

```

Reply:
Introduced `ReactionAppearanceType ` in 7d0da636ec9cdbc162fb1873d5428ee2a1d7e844:

```swift
/// The type describing message reaction appearance.
public protocol ReactionAppearanceType {
    var smallIcon: UIImage { get }
    var largeIcon: UIImage { get }
}
```

and updated `UIConfig` with `[MessageReactionType: ReactionAppearanceType]` in 7ab5481886074940bf325dd8b17328cc0157781d.

Used dictionary with reaction type as a key instead of array to make sure we don't have duplicates and avoid dealing with `Hashable`